zoo-css-challenge/style.css

```
@@ -118,17 +125,21 @@ footer {
 }
 
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+  background-color: #483C46;
+  color: #BEEE62;
+  font-size: 20em;
```

```suggestion
  font-size: 2em;
```
20em is awfully big! Make sure to test your changes